src/__tests__/types.dts.spec.ts

```diff
@@ -28,7 +28,7 @@ displayType<Immutable<User>>()
 // @dts-jest:pass:snap
 displayType<Immutable<undefined[]>>()
 // @dts-jest:pass:snap
-displayType<Immutable<null[]>>()
+displayType<Immutable<Array<null>>>()
```

Comment:
Doesn't looks to be relevant.

Reply:
It’ required due to the Typescript update.